Detectors/GlobalTracking/include/GlobalTracking/MatchTOF.h

```diff
@@ -243,110 +80,20 @@ class MatchTOF
-  std::unique_ptr<o2::gpu::GPUO2InterfaceRefit> mTPCRefitter;         ///< TPC refitter used for TPC tracks refit during the reconstruction
-
-  o2::dataformats::MCTruthContainer<o2::MCCompLabel> mTOFClusLabels;     ///< input TOF clusters MC labels
-  o2::dataformats::MCTruthContainer<o2::MCCompLabel>* mTOFClusLabelsPtr; ///< input TOF clusters MC labels (pointer to read from tree)
-  std::vector<o2::MCCompLabel> mTracksLblWork;                           ///<TPCITS track labels
-
-  gsl::span<const o2::MCCompLabel> mTPCLabels;  ///< TPC label of input tracks
-  std::vector<o2::MCCompLabel>* mTPCLabelsVect; ///< TPC label of input tracks (vector to read from tree)
-
-  gsl::span<o2::ft0::RecPoints const> mFITRecPoints; ///< FIT recpoints
-
-  /// <<<-----
-
-  ///<working copy of the input tracks
-  std::vector<matchTrack> mTracksWork;              ///<track params prepared for matching + time value
-  std::vector<float> mExtraTPCFwdTime;              ///<track extra params for TPC tracks: Fws Max time
-  std::vector<o2::track::TrackLTIntegral> mLTinfos; ///<expected times and others
-  std::vector<Cluster> mTOFClusWork;                ///<track params prepared for matching
-  std::vector<int8_t> mSideTPC;                     ///<track side for TPC tracks
-
-  ///< per sector indices of track entry in mTracksWork
-  std::array<std::vector<int>, o2::constants::math::NSectors> mTracksSectIndexCache;
-  ///< per sector indices of track entry in mTPCTracksWork
-  std::array<std::vector<int>, o2::constants::math::NSectors> mTPCTracksSectIndexCache;
-  ///< per sector indices of TOF cluster entry in mTOFClusWork
-  std::array<std::vector<int>, o2::constants::math::NSectors> mTOFClusSectIndexCache;
-
-  ///<array of track-TOFCluster pairs from the matching
-  std::vector<o2::dataformats::MatchInfoTOF> mMatchedTracksPairs;
-
-  ///<array of TOFChannel calibration info
-  std::vector<o2::dataformats::CalibInfoTOF> mCalibInfoTOF;
-
-  ///<array of matched TOFCluster with matching information (residuals, expected times...) with the corresponding vector of indices
-  //std::vector<o2::dataformats::MatchInfoTOF> mMatchedTracks;
-  std::vector<o2::dataformats::MatchInfoTOF> mMatchedTracks; // this is the output of the matching
-  std::vector<o2::MCCompLabel> mOutTOFLabels;                ///< TOF label of matched tracks
-
-  int mNumOfTracks;                     // number of tracks to be matched
-  std::vector<int> mMatchedTracksIndex; // vector of indexes of the tracks to be matched
-  int mNumOfClusters;                   // number of clusters to be matched
-  int* mMatchedClustersIndex = nullptr; //[mNumOfClusters]
-
   std::string mTracksBranchName = "TPCITS";                       ///< name of branch containing input matched tracks
   std::string mTPCTracksBranchName = "TPCTracks";                 ///< name of branch containing actual TPC tracks
   std::string mTPCMCTruthBranchName = "MatchMCTruth";             ///< name of branch containing TPC labels
```

Comment:
Do we really need this? I think TOF is the only detector which still maintains the no-DPL version of the reconstruction. There will be some effort for maintenance but what is the profit?

Reply:
I agree. I kept it to be consistent with the past.
If you also agree to remove it I am happy

Reply:
Yes, would be good to get rid of this legacy code.

Reply:
done